modules/tinymce/src/core/main/ts/init/Init.ts

```diff
@@ -94,6 +95,18 @@ const initTheme = (editor: Editor) => {
   }
 };
 
+const initModel = (editor: Editor) => {
+  const model = Options.getModel(editor);
+  const modelUrl = ModelManager.urls[model];
+  const Model = ModelManager.get(model);
+
+  try {
+    editor.model = Model(editor, modelUrl) || {};
+  } catch (e) {
+    ErrorReporter.modelInitError(editor, model, e);
+  }
```

Comment:
This doesn't seem like the correct behaviour, as if this fails to init the editor cannot continue unlike plugins. This seems like it would be better to follow the same pattern as the theme init logic instead where it just kills the load because an uncaught error is thrown.